docs/user_workloads/liveness_and_readiness_probes.md

```diff
@@ -29,172 +29,161 @@ operator and forwarded to the guest.
 
 ## Define a HTTP Liveness Probe
 
-The following VirtualMachineInstance configures a HTTP Liveness Probe
-via `spec.livenessProbe.httpGet`, which will query port 1500 of the
-VirtualMachineInstance, after an initial delay of 120 seconds. The
-VirtualMachineInstance itself installs and runs a minimal HTTP server on
+The following VirtualMachine configures a HTTP Liveness Probe
+via `spec.template.spec.livenessProbe.httpGet`, which will query port 1500 of the
+VirtualMachine, after an initial delay of 120 seconds. The
+VirtualMachine itself installs and runs a minimal HTTP server on
 port 1500 via cloud-init.
 
 ```yaml
 apiVersion: kubevirt.io/v1
-kind: VirtualMachineInstance
+kind: VirtualMachine
 metadata:
-  labels:
-    kubevirt.io/vm: vmi-fedora-vmi
-  name: vmi-fedora
+  name: example-http-liveness-probe
 spec:
+  runStrategy: Always
   template:
-    metadata:
-      labels:
-        kubevirt.io/domain: vmi-fedora
-        kubevirt.io/vm: vmi-fedora
-  domain:
-    devices:
-      disks:
-      - disk:
-          bus: virtio
-        name: containerdisk
-      - disk:
-          bus: virtio
-        name: cloudinitdisk
-      rng: {}
-    resources:
-      requests:
-        memory: 1024M
-  livenessProbe:
-    initialDelaySeconds: 120
-    periodSeconds: 20
-    httpGet:
-      port: 1500
-    timeoutSeconds: 10
-  terminationGracePeriodSeconds: 0
-  volumes:
-  - name: containerdisk
-    containerDisk:
-      image: quay.io/containerdisks/fedora:latest
-  - cloudInitNoCloud:
-      userData: |-
-        #cloud-config
-        password: fedora
-        user: fedora
-        chpasswd: { expire: False }
-        bootcmd:
-          - ["sudo", "dnf", "install", "-y", "nmap-ncat"]
-          - ["sudo", "systemd-run", "--unit=httpserver", "nc", "-klp", "1500", "-e", '/usr/bin/echo -e HTTP/1.1 200 OK\\nContent-Length: 12\\n\\nHello World!']
-    name: cloudinitdisk
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: containerdisk
+          - disk:
+              bus: virtio
+            name: cloudinit
+          rng: {}
+        resources:
+          requests:
+            memory: 1Gi
+      livenessProbe:
+        initialDelaySeconds: 120
+        periodSeconds: 20
+        httpGet:
+          port: 1500
+        timeoutSeconds: 10
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - name: containerdisk
+        containerDisk:
+          image: quay.io/containerdisks/centos-stream:9
+      - name: cloudinit
+        cloudInitNoCloud:
+          userData: |-
+            #cloud-config
+            password: centos
+            user: centos
```

Comment:
Why did you switch from Fedora to CentOS?

Reply:
Personal preference? - I had some existing samples which I re-jigged  - I have no issues changing that if there is some alignment to work towards, but from looking at the Kubevirt docs in total it seems not?

Reply:
All good. I just wanted to make sure there were no other deeper reasons. Thank you!